packages/worker/src/worker.ts

```diff
@@ -220,6 +220,9 @@ export interface WorkerOptions {
    * A mapping of interceptor type to a list of factories or module paths
    */
   interceptors?: WorkerInterceptors;
+
+  /** If set true, silence all `console.log` calls from workflow code */
+  silenceWorkflowLogging?: boolean;
```

Comment:
Instead, I'd let users override the console.log implementation.
I wouldn't want to expose this to users.